tox.ini

```diff
@@ -1,5 +1,6 @@
 [tox]
-envlist = py27
+envlist = py27, py34, py35, py36
+skip_missing_interpreters = true
```

Comment:
I don't think we should use this option. Wouldn't that cause CircleCI to pass tests even if it never tested some of the Python versions?